ServantHDBC/src/Api/Product.hs

```diff
@@ -0,0 +1,94 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeOperators #-}
+
+
+module Api.Product (ProductAPI, productHandlers) where
+
+import Control.Monad.Trans.Except
+import Data.Aeson
+import GHC.Generics
+import Servant
+
+
+data Product = Product
```

Comment:
Is this definition correct? This seems like the definition for `ProductFilter` and not `Product`